contracts/balance/contract.go

```diff
@@ -306,7 +464,7 @@ func (t Token) balanceOf(ctx storage.Context, holder interop.Hash160) int {
 	return acc.Balance
 }
 
-func (t Token) transfer(ctx storage.Context, from, to interop.Hash160, amount int, innerRing bool, details []byte) bool {
+func (t Token) _transfer(ctx storage.Context, from, to interop.Hash160, amount int, innerRing bool) bool {
```

Comment:
It MUST emit `Transfer` event per NEP-17.

Reply:
We talked about `TransferX`, that's a completely different thing that you can basically do whatever you want with. But `balance` is NEP-17 compatible so all token movements MUST produce `Transfer` event.